src/vanilla/Model/CodeModelTS.cs

```diff
@@ -393,7 +402,13 @@ public virtual string ConstructRuntimeImportForModelIndex()
 
         public virtual string PackageDependencies()
         {
-            return "\"ms-rest-js\": \"~0.22.422\"";
+            string deps = "\"ms-rest-js\": \"~0.22.422\"";
+            if (!string.IsNullOrEmpty(Settings.DefaultApiVersionPackage))
+            {
+                string version = Settings.AliasedNpmVersion ?? "^1.0.0";
```

Comment:
In GeneratorSettingsTS we call NPM to get the latest version number of different packages. I'd recommend doing that here. If the configuration has an `AliasedNpmVersion`, then use that, otherwise use whatever is latest in NPM. If that doesn't exist, then default to "^1.0.0".

Reply:
I can try to initialize the AliasedNpmVersion at the same time as the PackageVersion